docs/site/home/gremlin.html

```diff
@@ -18,17 +18,27 @@
         <meta charset=utf-8>
         <meta http-equiv="X-UA-Compatible" content="IE=edge">
         <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+        <meta name="robots" content="index, follow">
         <link href="img/favicon.png" rel="shortcut icon" />
-        <title>Apache TinkerPop: Gremlin</title>
-        <meta name="description" content="">
-        <meta name="keywords" content="">
-        <link href="css/bootstrap.min.css" rel="stylesheet">
-        <link href="css/font-awesome.css" rel="stylesheet"/>
-        <link href="css/custome.css" rel="stylesheet">
-        <link href="css/responsive.css" rel="stylesheet">
-        <link href="css/animate.css" rel="stylesheet"/>
-        <link href="css/prism.css" rel="stylesheet">
-        <link href="css/fonts.css" rel="stylesheet">
```

Comment:
Was there a reason these css links were removed?

Reply:
Thank you, @kmcginnes!  That was a miss in this file, added them.